### PR TITLE
Chris/refactor create utils

### PR DESCRIPTION
**Pull request by chrisong88**
NOTE: don't check this pull request until chris/api-refactor is merged. Because this branch is built off of that branch. (We might have to rebase; I'm not entirely sure)

Moved most helper functions to a utils/ folder. Could not move zrp_feature_engineering because it creates errors when loading the pickle file. I think that the pickle file has its own import statements encoded and expects the helper functions to be in specific locations. Need to check with Rakesh.

Potential issue to flag: I'm not entirely sure if I set up the absolute imports of the helper modules properly. I defined the absolute import paths in the context of src/ being the root directory of the app (because that's where app.py runs from).

### CODE REVIEW

**Review by rnori-harv — APPROVED** [review]
Great work on the modularization with utils! Just pulled and the functionality seems to still hold, so I think we're good! As for the zrp name and endpoint, we should probably change its name to zrpFlorida as we will be integrating NC next, but this is a minor stylistic change.

File: src/app.py [hunk 1]
```diff
@@ -29,37 +29,21 @@ def say_hello_world():
     return {'result': "Flask says Hello World"}
 
 @app.route('/surgeo', methods=["GET", "POST"])
```
Comment (dzheng21) [inline src/app.py:31]: These make sense for now, seems like we'll make another route in the future that goes something like: 
`@app.route('/prediction-comparison', methods=['POST']
x = dataAugmentation(data)
return run_surgeo(x) and zrp(x)`

File: src/app.py [hunk 2]
```diff
@@ -29,37 +29,21 @@ def say_hello_world():
...
-    zip_series = pd.Series([zipcode])
-
-    sg = surgeo.SurgeoModel()
-    sg_results = sg.get_probabilities(surname_series, zip_series)
 
-    sg_json = pd.DataFrame.to_json(sg_results)
-    return(sg_json)
-
-    # END OF SURGEO API; END OF TO-DO @KAYLA
+    return run_surgeo(surname=surname,
```
Comment (dzheng21) [inline src/app.py]: nitty gritty thing — `run_surgeo` vs. just `zrp`? either take out or add `run_` prefix

**Review by dzheng21 — APPROVED** [review]
looks good, minor changes and can merge

File: src/app.py (hunk 2 above)
Comment (chrisong88) [reply src/app.py]: Will do. run_surgeo was because "surgeo" is a protected keyword (the name of the actual surgeo module)

**Pull request merged**